### ISSUE TITLE
Does Airelle ever fix these? Another major legit (and very well known) ANGRYIP.ORG (the scanner EVERYONE uses) made his HIGH-RISK list: https://v.firebog.net/hosts/Airelle-hrsk.txt

### ISSUE THREAD

**Issue by CollinChaffin** [WaLLy3K/wally3k.github.io#58]
Airelle never replies to me but looks like he at least occasionally does for you.  I hate to add to others having you be the one to have to reach out, but perhaps add it to your queue for next time you do I think you'll agree that angry ip scanner has been around FOREVER as pretty much one of a few go-to freeware network scanners so I was a bit shocked to see this domain is now listed and on the "high-risk" no less at `https://v.firebog.net/hosts/Airelle-hrsk.txt`.

**Comment by WaLLy3K** [comment]
Hey @CollinChaffin,

Thanks for taking the time to reach out, it's always good to hear feedback from others about their experiences with particular blocklists.

It's rather frustrating to hear that Airelle doesn't reply to yourself and others, and while it's currently the holiday season (where one can reasonably expect delayed replies), it doesn't excuse the fact that I've been hearing about this for at least last six months now.

All I can do at this point is all I've already done: put the list as an unrecommended list (X), so people using [tick lists](https://v.firebog.net/hosts/lists.php?type=tick) don't run into these issues. I'd definitely recommend updating your blocklist to remove Airelle's content!

If there's anything else I can do, you're absolutely welcome to let me know - I'm always open to suggestions :smile:

**Comment by CollinChaffin** [comment]
Hey @WaLLy3K sorry I didn't reply sooner and thanks for the reply!  I couldn't agree more.  I have done more research on this since opening this issue, and to give you a good chuckle if you haven't already noticed this just take a look at the very first line in this Airelle list I also had active - one of the top `Malicious Lists` category we REALLY all need to watch out for:

LIST:  https://v.firebog.net/hosts/Airelle-hrsk.txt
CATEGORY:  Malicious Lists
Entry number 1 - be sure to blacklist this one, LOL!:

![snag_1-16-2019_08-45-03a](https://user-images.githubusercontent.com/9301483/51258941-bda13100-1970-11e9-9622-8644957bd7a3.png)


It looks like pi-hole recent revs will regardless not add if not a FQDN, and yes I'm sure this is just due to automated host file format generation etc. but it really is very eye-opening to actually open the Airelle lists and scan through them to see how many valid sites they seem to contain these days.

Just this AM, only after the Airelle list allowed me to navigate to `freewarefiles.com` to update a well known and SAFE utility hosted there, I also found that apparently Airelle knows that much better than the rest of us to have the certainty to decide that ALL freeware on that site is also unsafe MALWARE, so has graciously added `download.freewarefiles.com`, just to ensure we can be teased and allowed to look - but not actually download.  In reality, in all the years I've used that site as much as it's certainly a possibility a DEV could post something malicious, I don't EVER remember getting anything I would categorize as malware from it. Yes, it's just another whitelist, but what is easier, remove those lists, or leave them active and have to whitelist half the Internet? :)

To anyone else having issue, the very FIRST thing I would do is remove or at least disable ALL Airelle lists and stick to Wally's other lists, since the Airelle lists seem to not only be becoming more and more inaccurate, but it seems more and more impossible these days to even get erroneous entries corrected.

**Comment by beerisgood** [comment]
@WaLLy3K the localhost still exist in your https://v.firebog.net/hosts/Airelle-hrsk.txt copy.
Maybe just remove it?

Project's like https://github.com/nextdns/metadata use this list

**Comment by WaLLy3K** [comment]
> @WaLLy3K the localhost still exist in your https://v.firebog.net/hosts/Airelle-hrsk.txt copy.
> Maybe just remove it?
> 
> Project's like https://github.com/nextdns/metadata use this list

Thanks @beerisgood, I've added `localhost$` (Regex, so it'll match any line that ends with `localhost`) to the list of lines to remove in my update script. I've forced [HRSK](https://v.firebog.net/hosts/Airelle-hrsk.txt) to update, which now shows that line removed.

**Comment by p1r473** [comment]
He is blocking hackerone.com too under highrisk
